assets/js/app.js

```diff
@@ -102,23 +54,24 @@ function readyQuiz() {
   $('.questions-total').empty().append('4');
 }
 
-/* ---------- c) restartQuiz ---------- */
+/* ----------- c) restartQuiz ---------- */
 
 function restartQuiz() {
   $('.restart-button').on('click', function(event) {
     event.preventDefault();
     myQuiz.reset($('.results-page'), $('.questions-page'));
+    initLoad();
```

Comment:
This is going to attach a new event handler to $('.answers-btn') each time you restart. To test, try restarting the quiz a couple times, and adding a console log on line 40. You'll see that console.log get called for each time you've restarted.